IUPAC_SMILES+.asciidoc

```diff
@@ -102,7 +109,7 @@ http://www.daylight.com/meetings/mug03/McLeod/MUG03McLeodPeters.pdf[McLeod Peter
 ****
 
 McLeod and Peters' quip captures the deficiencies of SMILES well: if you
-*can't* build a molecule from a modeling kit, the deficiencies of SMILES and other
+*can not* build a molecule from a modeling kit, the deficiencies of SMILES and other
```

Comment:
I am not a native English speaker, but ['cannot' seems to be more common form](https://en.wiktionary.org/wiki/cannot#Usage_notes). Moreover, the specification document already uses 'cannot' form.